lib/helpers/remove-configuration-html-comments.js

```diff
@@ -4,7 +4,7 @@
 //   <!-- template-lint no-triple-curlies=false -->
```

Comment:
I can't seem to find any usage of this file.

Reply:
Hmm, can you check the git history to try to understand what it was used for? If no longer used/needed, you could open a PR to delete it.

Reply:
Was first added here:
https://github.com/ember-template-lint/ember-template-lint/commit/b4fa74127b832e9501e2735329aef3ae80b16a45#diff-01fba1b817dac0c337420c020783c2551254b949bb36779dec379af485781f80

And it seems, the last usage was deleted here:
https://github.com/ember-template-lint/ember-template-lint/commit/85c99325d3fc164dc541735980da3d5258b17cc6 (Line 33 of the deleted `index.js` file)

Can't find any further usage of this file starting from `v0.5.0-alpha.1`.